build-system/tasks/runtime-test.js

```diff
@@ -57,16 +57,33 @@ function getConfig() {
   return extend(obj, karmaConfig.default);
 }
 
-function getAdExtensions() {
-  const adNetworks = [];
+function getAdTypes() {
+  const namingExceptions = {
```

Comment:
If 3P used a different naming and forgot to update the mapping here, the test will fail.
Basically this is a place for exemptions.